src/slurm_plugin/clustermgtd.py

```diff
@@ -660,6 +713,24 @@ def _handle_health_check(
                             unhealthy_node.nodeaddr,
                         )
                         self._static_nodes_in_replacement.remove(unhealthy_node.name)
+                        if self._is_protected_mode_enabled():
+                            self._handle_bootstrap_failure_nodes([unhealthy_node])
+                            log.warning(
+                                "Node bootstrap error: Node %s(%s) failed health check during replacement, "
```

Comment:
Minor: `SlurmNode` object has a defined `__str__` function, so we can probably just do `"%s", node` directly as you are doing in other places and the class will print `f"{self.name}({self.nodeaddr})"` automatically.

Reply:
Done